opentracing/tracer.py

```diff
@@ -46,25 +79,42 @@ def start_span(self,
                    child_of=None,
                    references=None,
                    tags=None,
-                   start_time=None):
-        """Starts and returns a new Span representing a unit of work.
+                   start_time=None,
+                   ignore_active_span=False):
+        """Starts and returns a new Span representing a unit of work,
```

Comment:
I'd like to make sure that `start_active_span` seems like the default / preferred option, but I don't want us to break code that relies on `start_span` now... maybe something like `start_manual_span` to follow Java's `startManual()`?